lib/dataTrigger/dpnsTriggers/domainCreateDataTrigger.js

```diff
@@ -0,0 +1,63 @@
+const DataTriggerExecutionResult = require('../DataTriggerExecutionResult');
+const DataTriggerExecutionError = require('../../errors/DataTriggerExecutionError');
+const hash = require('../../util/hash');
+const Document = require('../../document/Document');
+const DataTrigger = require('../DataTrigger');
+
+/**
+ * @typedef domainCreateDataTrigger
+ * @param {Document} document
+ * @param {DataTriggerExecutionContext} context
+ * @return {Promise<DataTriggerExecutionResult>}
+ */
+async function domainCreateDataTrigger(document, context) {
```

Comment:
`createDomainDataTrigger` ?

Reply:
Why? This file exports an instance of `DataTrigger`, not a function

Reply:
This function is the trigger callback

Reply:
Ah, you mean just rename it, not make it a factory?